docs/MAIN_CONTROLLER.md

```diff
@@ -0,0 +1,29 @@
+## Main Controller
+### Front Panel
+
+![Controller - Front Panel](imgs/manual/front_panel_main.png)
+
+- **DISPLAY**: Show IP Address, Hostname, and SSH Password.
+- **ON/STANDBY**: Show state of the audio controller:
+    - Green: Unit is on
+    - Red: Unit is in Standby
+    - Blinking Red: Unit is waiting to be configured
```

Comment:
This sounds like its waiting for updates or something, I would say is in the process of booting?